hist/hist/src/HFitImpl.cxx

```diff
@@ -379,10 +382,10 @@ TFitResultPtr HFit::Fit(FitObject * h1, TF1 *f1 , Foption_t & fitOption , const
       f1->SetNumberFitPoints(fitdata->Size() );
 
       assert((Int_t)fitResult.Parameters().size() >= f1->GetNpar() );
-      f1->SetParameters( const_cast<double*>(&(fitResult.Parameters().front()))); 
-      if ( int( fitResult.Errors().size()) >= f1->GetNpar() ) 
-         f1->SetParErrors( &(fitResult.Errors().front()) ); 
-  
+      f1->SetParameters( const_cast<double*>(&(fitResult.Parameters().front())));
```

Comment:
const_cast-ing usually means that we are missing an abstraction or there is an implementation bug.